src/openfl/_internal/renderer/opengl/GLShaderManager.hx

```diff
@@ -56,7 +56,7 @@ class GLShaderManager extends AbstractShaderManager {
 	
 	public override function setShader (shader:Shader):Void {
 		
-		if (currentShader == shader) return;
+		// if (currentShader == shader) return;
```

Comment:
This is a temporary hack because the batcher changes the shader without the ShaderManager knowing.